docs/root/intro/what_is_envoy.rst

```diff
@@ -75,8 +75,8 @@ for DynamoDB connections.
 
 **Service discovery:** :ref:`Service discovery <arch_overview_service_discovery>` is a critical
 component of service oriented architectures. Envoy supports multiple service discovery methods
-including asynchronous DNS resolution and REST based lookup via a :ref:`service discovery service
-<arch_overview_service_discovery_types_sds>`.
+including asynchronous DNS resolution and REST based lookup via a :ref:`endpoint discovery service
```

Comment:
Can you update this section to just talk about dynamic config discovery in general and link to the dynamic configuratoin arch overview page?

Reply:
Done (merged with the 'dynamic configuration' item below).